test/luaunit/utility_test.lua

```diff
@@ -580,7 +580,10 @@ function g.test_parse_cmd_line()
     -- repeat
     assert_subject({'--repeat', '123'}, {exe_repeat=123})
     assert_subject({'-r', '123'}, {exe_repeat=123})
-    t.assert_error_msg_contains('Invalid value for -r option. Integer required', subject, {'-r', 'bad'})
+    t.assert_error_msg_contains('Invalid value for --repeat option. Positive integer required',
```

Comment:
tests don't check negative values :)

Reply:
Actually, it is not necessary because zero already covers this case.